### ISSUE TITLE
Consider supporting *_FILE convention for swarm/k8s secrets

### ISSUE THREAD

**Issue by decentral1se** [rocker-org/rocker-versioned2#159]
Swarm deployments use [secrets](https://docs.docker.com/engine/swarm/secrets/) to avoid exposing secrets in env vars and this means there is a growing convention to allow to provide a `*_FILE` matching env var so that deployments can specify files where secrets are stored and have them loaded. 

See for example:

![image](https://user-images.githubusercontent.com/1991377/118498088-f2cb1300-b725-11eb-8d3d-32543f268756.png)

(from https://hub.docker.com/_/mariadb/)

So, can we have a `PASSWORD_FILE` env var for passing the password for the rstudio user?

Thanks for your time :wave:

**Comment by cboettig** [comment]
yeah, interesting idea.  Sounds like this would apply only to  `swarm` users though, users running stand-alone docker containers or using a different orchestration/deployment system would need to stick with the current approach (?).  Would be open to a PR for this, would probably be a relatively straight-forward edit to the user config scripts run at startup.

**Comment by decentral1se** [comment]
> would need to stick with the current approach (?)

It would be optional for all deployments outside of k8s/swarm. If you need `_FILE` its there otherwise, just use `PASSWORD`.

> Would be open to a PR for this, would probably be a relatively straight-forward edit to the user config scripts run at startup.

Wonderful! Will get a PR together when I get back to this.
